### PR TITLE
feat: consolidate nightly workflows with shared skills and triage gate

### PR DESCRIPTION
**Pull request by CodySwannGT**
## Summary

- Extract analytical triage logic into a shared `ticket-triage` skill that gates the jira-agent (blocks work on ambiguous tickets) and replaces the 200-line inline prompt in the nightly JIRA triage workflow
- Create nightly execution skills (`nightly-add-test-coverage`, `nightly-lower-code-complexity`, `nightly-improve-tests`) and slim all 3 nightly workflow prompts down to skill invocations
- Create `plan-improve-tests` skill and `/plan:improve-tests` command for interactive test quality improvement
- Extract the security audit handling block (duplicated in 7 workflows) into an auto-loaded rule
- Add "Improve: Test Quality" sub-flow to intent routing

### New skills (source: `plugins/src/base/skills/`)
- `ticket-triage` — analytical triage gate (ambiguity detection, edge case analysis, verification methodology)
- `nightly-add-test-coverage` — direct-execution for nightly coverage improvement
- `nightly-lower-code-complexity` — direct-execution for nightly complexity reduction
- `nightly-improve-tests` — direct-execution for nightly test quality (nightly + general modes)
- `plan-improve-tests` — interactive plan skill for test improvement

### New commands
- `/jira:triage <TICKET-ID>` — run analytical triage on a ticket
- `/plan:improve-tests <target>` — improve test quality

### New rule
- `security-audit-handling.md` — auto-loaded rule for handling GHSA/bundler-audit failures during git push

### Modified agents
- `jira-agent` — added Step 3 (Analytical Triage Gate) between validation and intent determination

### Modified workflows
- `reusable-claude-nightly-jira-triage.yml` — replaced ~110 lines of inline logic with skill invocation
- `reusable-claude-nightly-test-coverage.yml` — replaced ~30 lines with skill invocation
- `reusable-claude-nightly-code-complexity.yml` — replaced ~25 lines with skill invocation
- `reusable-claude-nightly-test-improvement.yml` — replaced both prompts with skill invocations
- `reusable-claude-ci-auto-fix.yml` — removed inline security audit block
- `reusable-claude-code-review-response.yml` — removed inline security audit block
- `reusable-claude-deploy-auto-fix.yml` — removed inline security audit block

### Modified rules
- `intent-routing.md` — updated JIRA Entry Point (triage gate), added Improve: Test Quality sub-flow

## Test plan

- [x] `bun run build:plugins` — all new skills/commands propagate to `plugins/lisa/`
- [x] YAML validation passes on all 7 modified workflows
- [x] `grep -r "SECURITY AUDIT" .github/workflows/` returns no matches in TS workflows
- [x] All pre-push checks pass (lint, typecheck, tests, security audit, knip)
- [ ] Trigger nightly workflows via `workflow_dispatch` on a test repo
- [ ] Run `/jira:triage PROJ-123` interactively to verify skill produces structured output

🤖 Generated with Claude Code

## Summary by CodeRabbit

* **New Features**
  * Analytical JIRA triage gate before work starts
  * New skills for test quality, test coverage, complexity reduction, planning, triage, and related flows
  * New high-level commands: build, fix, improve, investigate, monitor, plan, review, ship

* **Documentation**
  * Major docs overhaul to describe governance/flow model and many new skill guides
  * CI/workflow prompts simplified—explicit security-audit remediation steps removed; separate security-audit handling doc added

* **Chores**
  * Plugin version metadata bumps

### CODE REVIEW

File: .github/workflows/reusable-claude-nightly-jira-triage.yml [hunk 1]
```diff
@@ -208,15 +151,15 @@ jobs:
...
-            ## Step 6: Output summary
+            ## Step 5: Output summary
 
             After processing all tickets, output a summary:
             - Total tickets processed
-            - For each ticket: key, summary, relevance (relevant/skipped), number of ambiguities found, number of edge cases found, whether verification methodology was added
+            - For each ticket: key, summary, verdict, number of ambiguities found, number of edge cases found, whether verification methodology was added
           claude_args: |
-            --allowedTools "Read,Glob,Grep,Bash(*)"
+            --allowedTools "Read,Glob,Grep,Bash(*),Skill(*)"
```
[the remark this answers is not in the document]
Comment (CodySwannGT) [reply .github/workflows/reusable-claude-nightly-jira-triage.yml]: This is intentional
